contrib/install.sh

```diff
@@ -0,0 +1,1187 @@
+      interactive_setup
+      return 1
+    fi
+
+    # Validate filters format
+    if [[ "$filters_start_height" =~ $filters_start_height_regex ]]; then
+      enable_cfilters=true
+      dialog --title "Floresta-Installer (Set enable filters)" --msgbox "✅ Enable filters set successfully: $filters_start_height" 10 60
+      interactive_setup
+      return 0
+    else
+      enable_cfilters=false
+      dialog --title "Invalid filter start height" --msgbox "❌ The filters start height you entered is invalid.\n\nIt should be in the format:\n- Positive number (1, 2, 3...)\n- Negative number (-1, -2, -3...)\n\nReturning to setup menu." 12 60
+    fi
+  done
+}
+
+# func: interactive_ask_assume_valid
+#
+# Ask if user wants to use --assume-valid=<BLOCK_HASH> with validation
+interactive_ask_assume_valid() {
+  if dialog --title "Floresta-Installer" --yesno "Do you want to use 'assume-valid'?\n\nThis option assumes that scripts before this height are validb." 15 60; then
+    interactive_assume_valid
+    if [ $? -ne 0 ]; then
+      assume_valid=""  # Clear invalid value
+      dialog --title "Floresta-Installer" --msgbox "❌ Invalid BLOCK_HEIGHT detected. Must be a numeric value." 8 45
+      return 1  # Restart process
+    fi
+    
+    # Show confirmation
+    dialog --title "Floresta-Installer" --msgbox "Assume-valid block height set to: $assume_valid" 8 45
+    interactive_setup
+  else
+    assume_valid=""
+    interactive_setup
+  fi
+}
+
+# func: interactive_assume_valid
+#
+# Prompt user to enter a valid BLOCK_HEIGHT for --assume-valid
+interactive_assume_valid() {
+  block_height=$(dialog --title "Floresta-Installer (Assume Valid)" \
+    --inputbox "Enter the BLOCK_HEIGHT you want to assume as valid (numeric value):" 10 60 \
+    3>&1 1>&2 2>&3)
+  
+  # Validate numeric input
+  if [[ "$block_height" =~ ^[0-9]+$ ]]; then
+    assume_valid="$block_height"
+    return 0
+  else
+    return 1
+  fi
+}
+
+# func: interactive_assume_utreexo
+#
+# Ask if user want to use --assume-utreexo or not
+interactive_assume_utreexo() {
+  if dialog --title "Floresta-Installer" --yesno "Do you want to use 'assume-utreexo'?\n\nThis option will significantly speed up the initial block download, by skipping the validation of the first hundreds of thousands of blocks. However, there's an inherent trust in the developer that the utreexo state is correct. Everything after the assumed height will be fully validated." 15 60; then
+    assume_utreexo=true
+  else
+    assume_utreexo=false
+  fi
+  interactive_setup
+}
+
+# func: interactive_ssl
+# 
+# Ask if user want to use SSL (Secure Sockets Layer) and if yes,
+# add --ssl-electrum-address, --ssl-cert-path and --ssl-key-path to florestad process.
+interactive_ssl() {
+  if dialog --title "Floresta-Installer" --yesno "Do you want to use SSL (Secure Sockets Layer)?" 8 45; then
```

Comment:
> Do you want to use SSL (Secure Sockets Layer) for the Electrum Server?

> This is particularly important if you want to access the Electrum Server from a public untrusted network. This will add encryption and authentication to your service.